README.md

```diff
@@ -12,38 +11,35 @@ Before starting the practical, make sure you have the following ready, if not, p
 * The URL [link](https://learning.ds-wizard.org/) for DSW training instance.
 * The [example scenario](https://github.com/elixir-luxembourg/DS-DM-training/blob/master/resources/DM-DP_RunningExample.pdf) for our workshop.
 
-## Exercice 1 : sign up
+## Exercice 1: sign up
 
 * Go to the url [link](https://learning.ds-wizard.org/) for DSW training and `Sign up`.
 
-
 ## Exercise 2: Create questionnaire (project) from existing knowledge model
 
 In the DSW tool, _Knowledge Models_ act as source databases that hold requirements in one or more sub-area of RDM.  
 The tool comes pre-packed with a comprehensive default knowledge model. In this practical we will use a cut-down version of this model. To create and fill in a questionnaire:
 
-
 1. Login to the DSW.<br/>
-2. On the left-hand side navigation menu click `Knowledge Models` to list models available. You should see `DM-DS Workshop Model` as below: <br/> <br/><br/>![Alt](images/list_workshop_km.png)<br/><br/><br/>
+2. On the left-hand side navigation menu click `Knowledge Models` to list models available. <br/> <br/><br/>![Alt](images/list_workshop_km.png)<br/><br/><br/>
 3. Click on the knowledge model named `DM-DS_Workshop_Model` to go to its overview page, which displays model's version history and available actions.<br/>
 4. Click `Create Project` on top right to create a project as a questionnaire from the workshop model: <br/><br/><br/>![Alt](images/project.png)<br/><br/><br/>
-5. Provide a name for your questionnaire, try to use a name that can distinguish your record from other trainees, for example include your user name in the questionnaire name. In addition, you may share your questionnaire with others. You may allowing either DSW tool users or anyone with a link to view/edit your answers. Clicking `Save` will create your questionnaire. <br/><br/><br/> ![Alt](images/project_name.png)<br/><br/><br/>
+5. Provide a name for your questionnaire, try to use a name that can distinguish your record from other trainees, for example include your user name in the questionnaire name. In addition, you may share your questionnaire with others. You may allow either DSW tool users or anyone with a link to view/edit your answers. Clicking `Save` will create your questionnaire. <br/><br/><br/> ![Alt](images/project_name.png)<br/><br/><br/>
```

Comment:
Is it possible to give them clear instructions how they should name their questionnaire? Do they receive some code or nickname for the training? 
Some people might fill in just gibberish and its gonna be impossible to asses their activities

Reply:
this time if students have their own account, they can rename the project as follows: username_projectName.

Reply:
Thank you i will put it